field_utils/undo_function_overload.macro

```diff
@@ -1,4 +1,4 @@
-#undef _FUNCN
+#undef _FUNC
```

Comment:
I thought this file had been deleted?

Reply:
The file is used by 'FieldBinaryOperatorTemplate.H'. The only reason it shows up for this PR is that I fixed the typo above. As it turns out, `FieldBinaryOperations.F90` undefines _FUNC, so I don't think that fixed line needs to exist, but I didn't delete it.